api/src/routes/protected/user.ts

```diff
@@ -433,6 +466,10 @@ export const userGetRoutes: FastifyPluginCallbackTypebox = (
       schema: schemas.getSessionUser
     },
     async (req, res) => {
+      const logger = fastify.log.child({ req });
+      // This is one of the most requested routes. To avoid spamming the logs
+      // with this route, we'll log requests at the debug level.
+      logger.debug('GET /user/get-session-user');
```

Comment:
Would we not be better with:

```suggestion
      logger.debug({ userId: req.user?.id });
```

Otherwise, we end up with:

```bash
DEBUG: GET /user/get-session-user {"req":{"method":"GET","url":"/user/get-session-user"}}
```